src/core/socket.c

```diff
@@ -794,12 +794,11 @@ static void socket_close_fds(Socket *s) {
         assert(s);
 
         LIST_FOREACH(port, p, s->ports) {
+                bool rm;
```

Comment:
yeah, will fix up the comment. Basically it should say that "we normally don't delete it, because … But if the use explicitly tells us to, we'll do it anyway"